### ISSUE TITLE
Private Window: Text highlight in address bar has almost the same color as the background

### ISSUE THREAD

**Issue by NotoriousNico** [rafaelmardojai/firefox-gnome-theme#883]
**Describe the bug**
I'm using v133.1 of your theme, which comes with a new purple color for private windows.
When entering text in the URL bar and highlighting the text, the text highlighter has almost the same color as the background color of the address bar, making it very hard to see:

![image](https://github.com/user-attachments/assets/19ce2c98-a7a4-461e-b4e1-dda83128c9c7)
Here's how the address bar looks in a regular window:
![image](https://github.com/user-attachments/assets/81cc7812-24a7-4910-ad8a-0becb93b76ee)

**To Reproduce**

1. Open a private window
2. Type in a URL
3. Highlight the URL

**Setup information**
 - OS: Ubuntu 24.10
 - Installation method: Manual
 - Firefox version: 133.0.3
 - Theme version: 133.1
 - Optional features enabled: None

**Comment by rafaelmardojai** [comment]
Yeah, but that's how it would look in libadwaita if an app sets that purple as accent color and headerbar background. And looks fine IMO, I don't want to use a different purple in dark mode honestly.

**Comment by NotoriousNico** [comment]
I understand, but even though that's how it would look like in libadwaita, I still think it's a bad user experience, if you can't actually see which part of the text is highlighted and which part is not. The two colors are almost identical:

Background: #714a90
Highlight: #6c448c

Anyway, I respect your decision on this.

And by the way: Thank you for this great theme and all the work you put into this.

**Comment by underlinejakez** [comment]
I believe this should be reconsidered. GNOME and Adwaita apps follows WCAG/APCA to contrast colors. Using [Contrast](https://gitlab.gnome.org/World/design/contrast) it's clear that this shouldn't be the case
![image](https://github.com/user-attachments/assets/5f36d8d7-7018-4bd0-845c-998638ba5645)

**Comment by NotoriousNico** [comment]
@underlinejakez Thank you so much for this.

@rafaelmardojai Can you please reconsider my request and re-open this ticket?

**Comment by mkhl** [comment]
For want of a better color I added this to my `theme/colors/dark.css`, under the `&[privatebrowsingmode="temporary"]` line:

    --gnome-selection-bg: color-mix(in srgb, var(--gnome-view-color) 30%, transparent);
